src/content/glossary.tsx

```diff
@@ -0,0 +1,82 @@
+export const terms = {
+    affinity: {
+        definition:
+            "Measure of the strength of an interaction between two molecules. Generally, the more chemical bonds that form between two molecules, the stronger the affinity of those molecules for each other.",
+        link: "[Learn more]",
+    },
+    KD: {
+        definition:
+            "The equilibrium constant. The balance of products and reactants at equilibrium, defined mathematically as the ratio of the dissociation and association rate constants, which quantitatively measures affinity.",
+        link: "https://www.khanacademy.org/science/ap-chemistry-beta/x2eef969c74e0d802:equilibrium/x2eef969c74e0d802:magnitude-and-properties-of-the-equilibrium-constant/v/magnitude-of-the-equilibrium-constant",
+    },
+    equilibrium: {
+        definition:
+            "The state in which the amount of reactants and products of a reaction is not changing over time.",
```

Comment:
Nit: I read this weirdly the first time. Should this be rephrased? I wasn't sure if it was:
"**amount** of reactants and products of a reaction **is**"
or
"_amount of reactants_ **and** _products of a reaction_ **are"**